packages/hot-updater/src/utils/fingerprint/fingerprint.spec.ts

```diff
@@ -0,0 +1,25 @@
+import fs from "fs";
+import os from "os";
+import path from "path";
+import { beforeEach, describe } from "vitest";
+
+const tmpDir = path.resolve(os.tmpdir(), ".hot-updater/test");
+// const iosDir = path.resolve(tmpDir, "ios");
+
+describe.skip("Fingerprint", () => {
+  beforeEach(() => {
+    if (fs.existsSync(tmpDir)) fs.rmSync(tmpDir, { recursive: true });
+
+    const exampleExpo52 = path.resolve(process.cwd(), "../../examples/expo-52");
```

Comment:
https://www.npmjs.com/package/workspace-tools

https://microsoft.github.io/workspace-tools/functions/getPnpmWorkspaces.html

I think we can use this to find the monorepo paths.